dipy/align/tests/test_expectmax.py

```diff
@@ -21,76 +21,82 @@ def test_compute_em_demons_step_2d():
+    # Select arbitrary centers
+    c_f = np.asarray(sh) / 2
     c_g = c_f + 0.5
 
-    #Compute the identity vector field I(x) = x in R^2
+    # Compute the identity vector field I(x) = x in R^2
     x_0 = np.asarray(range(sh[0]))
     x_1 = np.asarray(range(sh[1]))
-    X = np.ndarray(sh + (2,), dtype = np.float64)
+    X = np.ndarray(sh + (2,), dtype=np.float64)
     O = np.ones(sh)
-    X[...,0]= x_0[:, None] * O
-    X[...,1]= x_1[None, :] * O
+    X[..., 0] = x_0[:, None] * O
+    X[..., 1] = x_1[None, :] * O
 
-    #Compute the gradient fields of F and G
+    # Compute the gradient fields of F and G
     grad_F = X - c_f
     grad_G = X - c_g
 
-    #The squared norm of grad_G to be used later
-    sq_norm_grad_G = np.sum(grad_G**2,-1)
+    # The squared norm of grad_G to be used later
+    sq_norm_grad_G = np.sum(grad_G**2, -1)
 
-    #Compute F and G
-    F = 0.5*np.sum(grad_F**2,-1)
-    G = 0.5*sq_norm_grad_G
-    delta_field =  G - F
+    # Compute F and G
+    F = 0.5 * np.sum(grad_F**2, -1)
+    G = 0.5 * sq_norm_grad_G
+    delta_field = G - F
 
-    #Now select an arbitrary parameter for $\sigma_x$ (eq 4 in [Vercauteren09])
+    # Now select an arbitrary parameter for
+    # $\sigma_x$ (eq 4 in [Vercauteren09])
     sigma_x_sq = 1.5
 
-    #Set arbitrary values for $\sigma_i$ (eq. 4 in [Vercauteren09])
-    #The original Demons algorithm used simply |F(x) - G(x)| as an
-    #estimator, so let's use it as well
+    # Set arbitrary values for $\sigma_i$ (eq. 4 in [Vercauteren09])
+    # The original Demons algorithm used simply |F(x) - G(x)| as an
+    # estimator, so let's use it as well
     sigma_i_sq = (F - G)**2
 
-    #Select some pixels to have special values
+    # Select some pixels to have special values
     np.random.seed(1346491)
-    random_labels = np.random.randint(0, 5, sh[0]*sh[1])
+    random_labels = np.random.randint(0, 5, sh[0] * sh[1])
     random_labels = random_labels.reshape(sh)
 
-    random_labels[sigma_i_sq==0] = 2 #this label is used to set sigma_i_sq == 0 below
-    random_labels[sq_norm_grad_G==0] = 2 #this label is used to set gradient == 0 below
+    # this label is used to set sigma_i_sq == 0 below
+    random_labels[sigma_i_sq == 0] = 2
+    # this label is used to set gradient == 0 below
+    random_labels[sq_norm_grad_G == 0] = 2
 
     expected = np.zeros_like(grad_G)
-    #Pixels with sigma_i_sq = inf
+    # Pixels with sigma_i_sq = inf
     sigma_i_sq[random_labels == 0] = np.inf
     expected[random_labels == 0, ...] = 0
 
-    #Pixels with gradient!=0 and sigma_i_sq=0
+    # Pixels with gradient!=0 and sigma_i_sq=0
     sqnrm = sq_norm_grad_G[random_labels == 1]
     sigma_i_sq[random_labels == 1] = 0
-    expected[random_labels == 1, 0] = delta_field[random_labels == 1]*grad_G[random_labels == 1, 0]/sqnrm
-    expected[random_labels == 1, 1] = delta_field[random_labels == 1]*grad_G[random_labels == 1, 1]/sqnrm
+    expected[random_labels == 1, 0] = delta_field[
```

Comment:
I think we shouldn't break the line just after the square bracket `[`. How about:
```
    expected[random_labels == 1, 0] = (delta_field[random_labels == 1] *
                                       grad_G[random_labels == 1, 0] / sqnrm)
    expected[random_labels == 1, 1] = (delta_field[random_labels == 1] *
                                       grad_G[random_labels == 1, 1] / sqnrm)
```